apps/omg_watcher/lib/omg_watcher/api/core.ex

```diff
@@ -0,0 +1,86 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.API.Core do
+  @moduledoc """
+  Module provides API for compose exit
+  """
+
+  alias OMG.Block
+  alias OMG.State.Transaction
+  alias OMG.Utxo
+  use OMG.Utils.Metrics
+  require Utxo
+
+  @decorate measure_event()
+  def compose_output_exit([%{txindex: _} | _] = txs, Utxo.position(_, txindex, _) = decoded_utxo_pos) do
+    if txs |> Enum.any?(&match?(%{txindex: ^txindex}, &1)) do
+      sorted_tx_bytes =
+        txs
+        |> Enum.sort_by(& &1.txindex)
```

Comment:
I have a filling that all callers (to this function) already provide `txs` already sorted...
However I'm not sure we can assume this fact as granted... OK, never mind - just thinking aloud ;)